wear/src/main/java/org/gdg/frisbee/android/GdgWatchFaceConfigListenerService.java

```diff
@@ -0,0 +1,66 @@
+package org.gdg.frisbee.android;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.api.GoogleApiClient;
+import com.google.android.gms.wearable.DataMap;
+import com.google.android.gms.wearable.MessageEvent;
+import com.google.android.gms.wearable.Wearable;
+import com.google.android.gms.wearable.WearableListenerService;
+
+import java.util.concurrent.TimeUnit;
+
+public class GdgWatchFaceConfigListenerService extends WearableListenerService
+    implements GoogleApiClient.ConnectionCallbacks, GoogleApiClient.OnConnectionFailedListener {
+    private static final String TAG = "WatchFaceConfigService";
+
+    private GoogleApiClient mGoogleApiClient;
+
+    @Override
+    public void onMessageReceived(MessageEvent messageEvent) {
+        super.onMessageReceived(messageEvent);
+
+        if (!messageEvent.getPath().equals(WearableConfigurationUtil.PATH_ANALOG)) {
+            return;
+        }
+
+        byte[] rawData = messageEvent.getData();
+        DataMap configKeys = DataMap.fromByteArray(rawData);
+        Log.d(TAG, "configKeys = [" + configKeys + "]");
+
+        if (mGoogleApiClient == null) {
+            mGoogleApiClient = new GoogleApiClient.Builder(this).addConnectionCallbacks(this)
```

Comment:
This is only used to log onConnected, onConnectionSuspended, and onConnectionFailed now. Should I remove it?